packages/editor-ui/src/components/FixedCollectionParameter.vue

```diff
@@ -32,15 +32,17 @@
 		</div>
 
 		<div v-if="parameterOptions.length > 0 && !isReadOnly">
-			<n8n-button v-if="parameter.options.length === 1" fullWidth class="add-option" @click="optionSelected(parameter.options[0].name)" :label="getPlaceholderText" />
-			<el-select v-else v-model="selectedOption" :placeholder="getPlaceholderText" size="small" class="add-option" @change="optionSelected" filterable>
-				<el-option
-					v-for="item in parameterOptions"
-					:key="item.name"
-					:label="item.displayName"
-					:value="item.name">
-				</el-option>
-			</el-select>
+			<n8n-button v-if="parameter.options.length === 1" fullWidth @click="optionSelected(parameter.options[0].name)" :label="getPlaceholderText" />
+			<div class="add-option" v-else>
```

Comment:
let's have the `v-else` as first attribute

Reply:
changed